src/hearings/hearings.module.ts

```diff
@@ -24,7 +25,10 @@ import { effects, reducers } from './store';
     SharedModule,
     FormsModule,
     ReactiveFormsModule,
-    ExuiCommonLibModule
+    ExuiCommonLibModule,
+    MatAutocompleteModule,
+    MatTabsModule,
+    MatInputModule
```

Comment:
You probably don't need this Mat** in the hearings module, as we assume all the dependencies should be in common-lib module.